src/scss/components/statistics.scss

```diff
@@ -0,0 +1,125 @@
+.statistics {
+  position: relative;
```

Comment:
Please rename statistics.scss to _statistics.scss as we are using this convention.

Reply:
Done ! ;)